pkg/cmd/roachtest/tests/restore.go

```diff
@@ -883,14 +883,14 @@ func (bs backupSpecs) backupCollection() string {
 		bs.storagePrefix(), bs.workload.fixtureDir(), bs.version, bs.backupProperties)
 }
 
-// getAOSTCmd returns a sql cmd that will return a system time that is greater
-// or equal than the start time of the incremental backups in the target backup
-// chain.
+// getAOSTCmd returns a sql cmd that will return a system time that is equal to the end time of
+// the bs.backupsIncluded'th backup in the target backup chain.
 func (bs backupSpecs) getAostCmd() string {
-	return fmt.Sprintf(`SELECT max(DISTINCT start_time) FROM [SHOW BACKUP FROM %s IN %s] LIMIT %d`,
+	return fmt.Sprintf(
+		`SELECT max(end_time) FROM [SELECT DISTINCT end_time FROM [SHOW BACKUP FROM %s IN %s] LIMIT %d]`,
```

Comment:
I think you want order by:
SELECT max(end_time) FROM [SELECT DISTINCT end_time FROM [SHOW BACKUP FROM %s IN %s] ORDER BY end_time LIMIT %d]
Maybe it works without but there are no guarantees.

Reply:
Done.